src/components/modals/AlertModal/AlertModal.js

```diff
@@ -0,0 +1,97 @@
+import React from 'react'
+import styled from 'styled-components/macro'
+import PropTypes from 'prop-types'
+
+AlertModal.propTypes = {
+  children: PropTypes.string,
+  handleOnOk: PropTypes.func,
+  handleOnCancel: PropTypes.func,
+}
+
+function AlertModal({ children, handleOnOk, handleOnCancel }) {
+  return (
+    <ErrorWrapper>
```

Comment:
I think it is not an ideal solution to have the modal as a normal "component" in the DOM, because it is unusual to have the child element block the screen. It might also cause problems with your z-index, because the z-index is relative to its parent element (as I can tell from painful experience). 
You could have a look at React Portals to render the modal on the top level of your app. 
https://reactjs.org/docs/portals.html

Reply:
Thanks for the hint. I changed it. Didn't noticed a different in behavior. But I think it's the better practicse. Funny thing I already watched a video about this topic on Youtube ;-)